warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -1124,37 +1125,37 @@ private void writeStats(Path[] jobDirectories) throws IOException {
         // Write out the metrics.
         // We are going to serialize the counters into a file in HDFS.
         // The context was set in the processKeyValues method below, and should not be null. We'll guard against NPE anyway
-        FileSystem fs = getFileSystem(seqFileHdfs);
-        RawLocalFileSystem rawFS = new RawLocalFileSystem();
-        rawFS.setConf(conf);
-        CompressionCodec cc = new GzipCodec();
-        CompressionType ct = CompressionType.BLOCK;
-        
-        Counters c = reporter.getCounters();
-        if (null != c && c.countCounters() > 0) {
-            // Serialize the counters to a file in HDFS.
-            Path src = new Path(File.createTempFile("MapFileLoader", ".metrics").getAbsolutePath());
-            Writer writer = SequenceFile.createWriter(conf, Writer.file(rawFS.makeQualified(src)), Writer.keyClass(NullWritable.class),
-                            Writer.valueClass(Counters.class), Writer.compression(ct, cc));
-            writer.append(NullWritable.get(), c);
-            writer.close();
+        try (FileSystem fs = getFileSystem(seqFileHdfs); RawLocalFileSystem rawFS = new RawLocalFileSystem()) {
```

Comment:
I see other FileSystem objects above that for which we could possibly do the same: lines 1048, 1056

Reply:
Now that I think about it.  The reason for not closing the FileSystem objects that is we wanted to keep those in the cache.  We are not creating any files with the "delete on exit" flag set, so perhaps we should not be closing any of the FileSystem objects.

Reply:
comment ignored?

Reply:
I must have somehow lost the fix in one of my rebases. It's been un-ignored.